be/src/storage/lake/txn_log_applier.cpp

```diff
@@ -148,6 +151,22 @@ class PrimaryKeyTxnLogApplier : public TxnLogApplier {
         return Status::OK();
     }
 
+    Status apply_alter_meta_log(const TxnLogPB_OpAlterMeta& op_alter_meta) {
+        DCHECK_EQ(_base_version + 1, _new_version);
+        switch (op_alter_meta.table_meta_type()) {
+        case TabletMetaTypePB::ENABLE_PERSISTENT_INDEX:
+            _metadata->set_enable_persistent_index(op_alter_meta.enable_persistent_index());
+
+            // Try remove index from index cache
+            // If tablet is doing apply rowset right now, remove primary index from index cache may be failed
+            // because the primary index is available in cache
+            // But it will be remove from index cache after apply is finished
+            _tablet.update_mgr()->index_cache().try_remove_by_key(_tablet.id());
```

Comment:
Handle the return value. If the return value can be ignored, use `(void)` to ignore it explicitly.